ts/client/coreAPI.ts

```diff
@@ -73,7 +73,7 @@ export class CoreAPI implements ICoreAPI {
         const contracts = genesis.getContracts(signer);
         const syncedPoint = new SyncedPoint(
             genesis.auxiliary.genesisEth1Block,
-            0
+            -1
```

Comment:
Setting this to an uninitialized state and letting the genesis event sync move it to 0 seemed correct to me.